src/components/reportPage/results-page.css

```diff
@@ -82,5 +92,18 @@
   .results-container {
     flex-direction: row;
     flex-wrap: wrap;
+    justify-content: center;
+  }
+
+  .video-result-container {
+    flex-direction: row;
+    justify-content: center;
+  }
+
+  .video-result-explanation {
+    margin: 0 4rem;
+    border: 0.06rem solid grey;
+    padding: 3rem;
+    max-width: 30rem;
```

Comment:
Why is there two of `video-result-explanation` and `video-result-container` in this file?